src/Sparrow/Json/JsonOperationContext.cs

```diff
@@ -35,9 +32,10 @@ public class JsonOperationContext : IDisposable
 
         private int _lastStreamSize = 4096;
 
-        public JsonOperationContext(UnmanagedBuffersPool pool)
+        public JsonOperationContext()
         {
-            Pool = pool;
+            _arenaAllocator = new ArenaMemoryAllocator();
+            _arenaAllocatorForLongLivedValues = new ArenaMemoryAllocator(16 * 1024);
```

Comment:
I think that the use case is somewhat different. And at any case, we're probably going to make the byte string allocator simpler by having it depend directly on the arena allocator, which should simplify the whole thing